File: squadmatch_app/lib/domain/models/option.dart
class Option {
  final String id;
  final String title;
  final String imageUrl;
  final String description;

  Option({
    required this.id,
    required this.title,
    required this.imageUrl,
    this.description = "",
  });

  factory Option.fromJson(Map<String, dynamic> json) {
    return Option(
      id: json['id'],
      title: json['title'],
      imageUrl: json['imageUrl'],
      description: json['description'] ?? "",
    );
  }
}

File: Backend/SquadMatch.Domain/Models/Option.cs
namespace SquadMatch.Domain.Models;

public class Option
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public Guid RoomId { get; set; }
    public Room? Room { get; set; }
    public string ExternalApiId { get; set; } = string.Empty;
    public string Title { get; set; } = string.Empty;
    public string ImageUrl { get; set; } = string.Empty;
    public string Description { get; set; } = string.Empty;
}
